.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Pull Request
+on:
+  push:
+    branches: ["**"]
+  pull_request:
+    branches: ["**"]
+  workflow_dispatch:
+
+env:
+  GOPATH: /opt/go
+  PATH: /opt/go/bin:/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin
+  GO_VER: 1.18.2
```

Comment:
It might be worth moving to a later Go version, either:
- If GO_VER is used as a complete Go version outside of this workflow, explicitly setting `1.18.6`; or
- If GO_VER is only used in this workflow, using either `1.18` or `'^1.18.6'`